CorpusSearch/Services/OpenDataLoader.cs

```diff
@@ -60,6 +62,34 @@ public static List<string> GetJsonPaths(LoadConfig lConfig)
                     : allFiles.ToList()
                 ;
         }
+
+        
+
+        public static async Task<List<(IDocument, DateTime)>> LoadLatestDocuments(WorkService workService)
+        {
+            var latestDocumentsFile = Path.Combine(
+                AppDomain.CurrentDomain.BaseDirectory,
+                "OpenData",
+                "newdocs.txt"
+            );
```

Comment:
comment an expected path above for brevity

Make a note on `newdocs.txt` is generated